### ISSUE TITLE
Excel: Labels for background text causes errors when converting .po to .mo

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#6229]
Hi,

reopened #6092:

When translators translate latest NVDA master strings 9ones with background pattern), Poedit and similar tools say validation has failed. Specifically, validation filaure is flagged for messages that involve pattern text, such as 25% gray and similar.

CC @ManshulBelani, @MichaelDCurran, @jcsteh 
Thanks.

**Comment by josephsl** [comment]
Technical: percent (%) character must be escaped. I'll write a PR to fix this. Thanks.

CC @ManshulBelani,

**Comment by jcsteh** [comment]
> Poedit and similar tools say validation has failed.

When you say "similar tools", what other tools has this been reported with?

> Technical: percent (%) character must be escaped.

There's no way to escape it. The way you've done it in #6230 will cause a double percent sign for English. This string isn't meant to be interpreted as a % formatted string. The question is whether there's some way to prevent Poedit from doing this.

**Comment by jcsteh** [comment]
We can prevent Poedit and xgettext from marking these as python-format strings (and thus trying to validate the percent sign) by placing an "xgettext:no-python-format" comment before affected strings. See [this thread on the Poedit Google Group](https://groups.google.com/forum/#!topic/poedit/f34rrvx7UT8) and [this section of the GNU gettext documentation](https://www.gnu.org/software/gettext/manual/html_node/c_002dformat-Flag.html).

**Comment by josephsl** [comment]
@jcsteh: Fixed from my end.

Try:
1. As noted, enter no:python format string into nvda.po manually (from my side).
2. Translate.
3. Try compiling.

Result: successfully compiled, no validation issue found.

Thanks.
